serviceerror/namespace_invalid_state.go

```diff
@@ -40,6 +40,16 @@ func NewNamespaceInvalidState(namespace string, state enumspb.NamespaceState, al
 	}
 }
 
+// NewNamespaceInvalidStatef returns new NamespaceInvalidState error with formatted message.
+func NewNamespaceInvalidStatef(namespace string, state enumspb.NamespaceState, allowedStates []enumspb.NamespaceState, format string, args ...interface{}) error {
```

Comment:
do we really need this one? the main constructor does a good job with messages...